dotnet/dynamodb/UpdateItemDataModel/UpdateItemDataModel.cs

```diff
@@ -0,0 +1,170 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. 
+// SPDX-License-Identifier: MIT-0
```

Comment:
Wrong license type (here and throughout).

Reply:
Dang, I though I had fixed this. It's now:

SPDX - License - Identifier: Apache - 2.0